test/sideinput-sink-e2e/testdata/sideinput_sink.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: user-defined-sink-test
+spec:
+  sideInputs:
+    - name: myticker
+      container:
+        image: "quay.io/shubhamdixit863/sideinput-sink-e2e-test:latest"
```

Comment:
Please build a new image under quay.io/numaio and update the tag before merging—same for below. Basically we don't include personal images in numaflow repo.